### DISCUSSION TITLE
GD 2.3.4

### DISCUSSION THREAD

**Discussion by pierrejoye** [libgd/libgd#805] [General]
Merged bug fixes from master to 2.3, plan to release 2.3.4 in the next couple of days. Any other bug or PR I may have forgotten? @cmb69 @vapier 

2.4 will be the focus then, preparing the road to 3.0.

**Comment by cmb69** [comment]
Sounds good! I don't think it should be a show-stopper if there are still unresolved bugs in 2.3; should be fine to fix these in 2.4.x.

**Comment by pierrejoye** [comment]
I merged all found coverity fixes as well. Finishing cleaning the ones
found in tests/, it found 3 new after i cleaned them all first  :)

**Reply by pierrejoye** [reply]
One left in that damned tiff thingie :)

I need to think about adding some helpers for assert, free used res on error etc. That's cumbersome as it is now, too many if/goto all around :)
